content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-nodeable-nodes.md

```diff
@@ -0,0 +1,259 @@
+---
+linktitle: Custom Nodeable Nodes
+title: Creating Custom Nodeable Nodes in Script Canvas
+description: Learn how to create custom Script Canvas Nodeable Nodes in Open 3D Engine (O3DE).
+weight: 100
+---
+
+This topic guides you how to create custom Script Canvas Nodeable Nodes step by step.
+
+You'll see the term _nodeable_ used throughout the O3DE source. 
+A nodeable can refer to both the node that appears in the Node Palette as a result of the AzAutoGen processing,
+and the mechanism by which a compiled Script Canvas graph can invoke C++ functions.
+
+## Prerequisite: Adding support for custom nodeable nodes to a Gem
+In your Gem's `CMakeLists.txt`, add a section for `AUTOGEN_RULES` and declare `Gem::ScriptCanvas` as build dependency.
+
+The precise place for this section will vary depending on how your Gem is configured. 
+However, we recommend that your Gem define a `STATIC` library to make the code available to both runtime and editor projects.
+
+As an example, here is partial definition of Gem's `CMakeLists.txt` that supports Script Canvas custom nodes with following required changes:
```

Comment:
If this set of changes is going to be "step 1" in this topic, then perhaps we should use the same example Gem throughout, instead of using a placeholder "MyGem" in this step, followed by the timer library in subsequent steps.

Reply:
Use StartingPointInput Gem as example here, as it is also referred in later section.
Timer library is from a ScriptCanvas Gem, which CMakeLists is quite complicated and not really a good example here